syntax = "proto3";

package prod_im_client;
import "common.proto";

service prod_im_client_service {
    rpc send_chat_msg (send_chat_msg_req) returns (send_chat_msg_res) {}
}

message send_chat_msg_req {
    string sender_id = 1;
    string receiver_id = 2;
    string chat_msg = 3;
}

message send_chat_msg_res {
    int32 result = 1;
}
